### DISCUSSION TITLE
How soon till LiteDB 5.0+ is supported?

### DISCUSSION THREAD

**Discussion by bcalco** [Zaid-Ajaj/LiteDB.FSharp#68] [Q&A]
I ask because I am eager to use the latest LiteDB with an FSharp project.

What new features of LiteDB 5+ are lost by using LiteDB.FSharp? 

What changes in LiteDB 5+ that breaks the current LiteDB.FSharp?

**Comment by humhei** [comment]
https://github.com/mbdavid/LiteDB/pull/1978

People can fixes it by themselves, 
And then release a new local nuget package

Then merge this PR https://github.com/Zaid-Ajaj/LiteDB.FSharp/pull/61

Personally i use it for serveral months now,
